test-env/src/main/kotlin/io/spine/protodata/test/InternalAccessRenderer.kt

```diff
@@ -41,7 +41,7 @@ public class InternalAccessRenderer : SoloRenderer<Java>(Java) {
     override fun render(sources: SourceFileSet) {
         val internalTypes = select<InternalType>().all()
         internalTypes.forEach { internalType ->
-            val path = internalType.name.qualifiedName().replace('.', File.separatorChar)
+            val path = internalType.name.qualifiedName.replace('.', File.separatorChar)
```

Comment:
```suggestion
            val path = internalType.qualifiedName.replace('.', File.separatorChar)
```
```suggestion
            val path = internalType.name.qualified.replace('.', File.separatorChar)
```

I don't know whether it is possible, but now it looks a bit wordy. But not critical.